p2p/testing/peerpool.go

```diff
@@ -39,29 +39,29 @@ func NewTestPeerPool() *TestPeerPool {
 	return &TestPeerPool{peers: make(map[discover.NodeID]TestPeer)}
 }
 
-func (self *TestPeerPool) Add(p TestPeer) {
-	self.lock.Lock()
-	defer self.lock.Unlock()
+func (tpp *TestPeerPool) Add(p TestPeer) {
```

Comment:
Please use `p` instead of `tpp`, and rename the argument here and below to `peer` from `p`.